doc/mesh_filter/meshes/hokuyo.dae

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i believe these meshes are included by default in gazebo, right?

Reply:
All the gazebo models and meshes are here - http://models.gazebosim.org/

Reply:
I don't see where hokuyo.dae is used in this tutorial, just the kinect.dae...